src/ontology/mondo.Makefile

```diff
@@ -963,6 +963,20 @@ update-%-mappings: $(TMPDIR)/new-exact-matches-%.owl
 		make NORM
 		mv NORM $(SRC)
 
+$(TMPDIR)/mondo-omim-genes.robot.owl:
+	wget "https://raw.githubusercontent.com/monarch-initiative/mondo-ingest/hgnc-template/src/ontology/external/mondo_genes.robot.owl" -O $@
```

Comment:
I think you asked in the HGNC PRs to re-name this file.

Reply:
Yes thanks, we wont merge this before the mondo ingest pr is merged!

Reply:
- [ ] @matentzn Update the `wget` w/ correct URL once [Mondo PR](https://github.com/monarch-initiative/mondo-ingest/pull/559) has been merged.

File has been renamed: `mondo-omim-genes.robot.owl`

I'm sure this is also on your radar to update it so it pulls from the `main` branch eventually.

Just thinking about this now, that necessitates that we merge that PR into `main` after merging it into `develop`, so I'll do the merging / build running a couple more times when we're ready for that.

Reply:
Hasn't been merged into `main` yet, but has been merged into `develop`.

Reply:
Once we hear from Sabrina when the alignment work is done, we can decide when to bring updates from mondo-ingest `develop` back into `main` and make sure the proper test builds are done.